pandas/_libs/tslibs/strptime.pyx

```diff
@@ -4,7 +4,6 @@ import time
 import locale
 import calendar
 import re
-import datetime
```

Comment:
I think can just cimport cpython.datetime as datetime to minimize changes no? Didn't we move away from the import pattern here already in #32459

Reply:
Good catch!

Somehow slipped away from me